_docs/tutorial/07-canary-release-with-istio.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Canary release with istio
+overview: Overview
+
+order: 07
+
+layout: docs
+type: markdown
+---
+{% include home.html %}\n# Canary Release with Istio
+In this learning module we will deploy a new version of the _reviews_ microservice again,
+this time with Istio enabled. We will release our new version to the `jason` user only (`jason` is our tester). It will allow the `jason` user to test the whole application end-to-end in production, with our new version.
+
+1. Let's specify a routing rule that all the production traffic will flow to the version _v1_ of all the microservices:
```

Comment:
rule that all the production traffic will flow to the version _v1_-> rule to send all production traffic to version _v1_ of